tests/webapp/api/test_performance_alerts_api.py

```diff
@@ -230,3 +230,16 @@ def test_alerts_post_insufficient_data(test_repository,
                            new_post_blob)
         assert resp.status_code == 400
         assert PerformanceAlert.objects.count() == 0
+
+
+def test_job_ids_validity(test_repository,
+                          test_perf_alert_summary,
+                          test_perf_signature, test_sheriff,
+                          alert_create_post_blob):
+    client = APIClient()
+    client.force_authenticate(user=test_sheriff)
+    new_post_blob = copy.copy(alert_create_post_blob)
+    new_post_blob['job_ids'] = "foo bar"
+    resp = client.post(reverse('performance-alerts-list'),
```

Comment:
A few things:

* This isn't the right endpoint to test, you want `performance-data-list` (this is defined in `treeherder/webapp/api/urls.py`)
* Correspondingly, you should put this test in `test_performance_data_api.py`. 
* Also you want to get here, not post. There's  no need to create a post blob or anything.
* You can use the webapp fixture here, no need to instantiate an APIClient (just be sure to pass `expect_errors` to it)